pkg/apis/pipeline/v1beta1/implicit_params_test.go

```diff
@@ -34,240 +34,223 @@ func TestImplicitParams(t *testing.T) {
 	ctx := context.Background()
 	cfg := config.FromContextOrDefaults(ctx)
 	cfg.FeatureFlags = &config.FeatureFlags{EnableAPIFields: "alpha"}
+	ctx = config.ToContext(ctx, cfg)
 
-	ctxs := []struct {
-		name string
-		ctx  context.Context
+	for _, tc := range []struct {
+		// Expect in and want to be the same type.
+		in   apis.Defaultable
+		want interface{}
```

Comment:
could you include a name to indicate the purpose of each test case? i think its one for a pipelinerun, one for a taskrun (and i think nothing more complicated than that?) but it wasnt easy to tell without reading through each test case